module/cyphersystem.js

```diff
@@ -784,26 +784,26 @@ function allInOneRollDialog(actor, pool, skill, assets, effort1, effort2, additi
   function applyToMacro(pool, skill, assets, effort1, effort2, additionalCost, additionalSteps, stepModifier, title, damage, effort3, damagePerLOE) {
     let cost = parseInt(additionalCost);
     let effort = parseInt(effort1) + parseInt(effort2) + parseInt(effort3);
-    if (stepModifier == "hindered") additionalSteps = parseInt(additionalSteps) * -1;
+    if (stepModifier.toLowerCase() == "hindered") additionalSteps = parseInt(additionalSteps) * -1;
```

Comment:
Why is this needed?

Reply:
Good question, I may have just outbound myself there.

Reply:
Refactored translation